tests/devices/i22/test_fswitch.py

```diff
@@ -0,0 +1,42 @@
+from typing import Any, Mapping
+from unittest import mock
+
+import pytest
+from ophyd_async.core import DeviceCollector, StandardReadable, set_sim_value
+
+from dodal.devices.i22.fswitch import FilterState, FSwitch
+
+
+@pytest.fixture
+async def fswitch() -> FSwitch:
+    async with DeviceCollector(sim=True):
+        fswitch = FSwitch("DEMO-FSWT-01:")
+
+    return fswitch
+
+
+async def assert_reading(
+    device: StandardReadable,
+    expected_reading: Mapping[str, Any],
+) -> None:
+    await device.stage()
+    reading = await device.read()
+    await device.unstage()
```

Comment:
Should: Does this have to be staged/unstaged for the test to work?

Reply:
No, that's a copy of my mistake from #431, fix in #474